sorting/stooge_sort.cpp

```diff
@@ -0,0 +1,43 @@
+/**
+ * @file
+ * @brief [Stooge sort implementation](https://en.wikipedia.org/wiki/Stooge_sort)
+ * in C++
+ * @details
+ * Stooge sort is a recursive sorting algorithm.
+ * It divides the array into 3 parts and proceeds to:
+ *	- sort first two thirds of the array
+ *	- sort last two thirds of the array
+ *  - sort first two thirds of the array
+ * It's time complexity is O(n^(log3/log1.5)), which is about O(n^2.7), 
+ * which makes it to be not the most efficient sorting algorithm 
+ * on the street on average. Space complexity is O(1).
+ */
+
+#include <vector>
+#include <iostream>
```

Comment:
Please add a one-line description of what the library/header is for (**see the example below**).

```c++
#include <iostream>    /// for IO operations
#include <cassert>    /// for assert
```

Reply:
Just added comments for includes